### ISSUE TITLE
Add alt text to images

### ISSUE THREAD

**Issue by paocorrales** [21centuryweather/21st-Century-Weather-Software-Wiki#145]
The wiki relies on screenshots a lot to explain topics; these should have alt text to be accessible to screen readers. 

This is a big task; we probably need to create sub-issues to tackle section by section.
